.github/workflows/integration-auth-tests.yml

```diff
@@ -0,0 +1,138 @@
+name: Integration Auth Tests
```

Comment:
this is no bueno. we can't keep adding ollama to every single integration test. it is _way_ too much overhead. Let's figure out what additional integration about auth you want tested and just test that bit.

Reply:
makes sense - added a more minimal distro without inference.